gslab_fill/tablefill.py

```diff
@@ -77,39 +77,73 @@ def parse_data(data):
     return tables    
     
 
-def insert_tables(args,tables):
+def insert_tables(args, tables):
+    if re.search('\.tex', args['template']):
+        lyx_text = insert_tables_latex(args, tables)
+    else:
+        lyx_text = open(args['template'], 'rU').readlines()
+        for n in range( len(lyx_text) ):
+            if lyx_text[n].startswith('name "tab:') or re.search('label{tab:', lyx_text[n]):
+                tag = re.sub("[\}\"\n]", "", lyx_text[n].split(':')[1]).lower()

```

Comment:
Yea, you can probably revert the changes to the base `insert_tables` function because I was trying to make the switches throughout, but then gave up and made a new function because there were more changes than anticipated.